crates/ruff_server/src/server/api/requests/execute_command.rs

```diff
@@ -148,3 +163,25 @@ fn apply_edit(
         },
     )
 }
+
+fn debug_information(session: &Session) -> String {
+    let path = std::env::current_exe()
+        .map(|path| format!("{}", path.display()))
+        .unwrap_or_else(|_| "<unavailable>".to_string());
+    format!(
+        r#"path = {path}
```

Comment:
nit: would "executable" be more useful than "path" as it's a path to the `ruff` executable?